docs/manuals/source/DocumentationStyleGuide/BareosSpecificFormatting/Release.rst

```diff
@@ -96,6 +96,10 @@ Output:
 
 This refers to :ref:`bareos-17.2.7`\ .
 
+When refering to a Bareos version,
+always use the version number with three components (e.g. 17.2.7).
+When just refering to the 17.2 release, replace the remaining components by 0, e.g. 17.2.0.
```

Comment:
Using the 0 as placeholder is ambigous. There could be a version 17.2.0

It would be clear to use 17.2.x or 17.2.*, 17.2.N

Reply:
I would prefer "17.2" or even "17" That's just as easy as pie.

Reply:
That is both not what has been decided in the development meeting from  2019-10-17.

Reply:
Removed this lines from the PR, to get it integrated. If required, this can be handled in a separate PR. Until then, everybody will continue to refer to Bareos versions by his own style. Currently used:
```
.. deprecated:: 19
.. deprecated:: 19.2
:sinceVersion:`18.2.0: TLS-PSK`
Bareos Version 18.2
Bareos-18.2
Version 18.2
bareos-14.2.x
bareos-17.2.7
before 18.2
:ref:`bareos-17.2.5`
version 18.2
versionchanged:: 18.2
```